overlord/configstate/configcore/interface.go

```diff
@@ -0,0 +1,99 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//go:build !nomanagers
+
+/*
+ * Copyright (C) 2025 Canonical Ltd
```

Comment:
But are we in the end expected to put the year or not? Pretty confused about this now 🙂